app/src/main/java/com/example/hungrywolfs/model/AppViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.hungrywolfs.model
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.hungrywolfs.network.*
+import kotlinx.coroutines.launch
+
+class AppViewModel : ViewModel() {
```

Comment:
AppViewModel? Is there only one? Or is this one used in HomeFragment? Please rename.